packages/mdc-linear-progress/foundation.ts

```diff
@@ -21,12 +21,12 @@
  * THE SOFTWARE.
  */
 
-import {getCorrectPropertyName} from '@material/animation/index';
+import {getCorrectPropertyName} from '@material/animation/util';
 import {MDCFoundation} from '@material/base/foundation';
 import {MDCLinearProgressAdapter} from './adapter';
 import {cssClasses, strings} from './constants';
 
-class MDCLinearProgressFoundation extends MDCFoundation<MDCLinearProgressAdapter> {
+export class MDCLinearProgressFoundation extends MDCFoundation<MDCLinearProgressAdapter> {
```

Comment:
Apparently not in this PR, but somehow this foundation lost the constructor which mixes the default adapter with the passed adapter.

Reply:
Good catch! Added.